src/ray/raylet/worker_pool.cc

```diff
@@ -152,8 +152,19 @@ void WorkerPool::SetAgentManager(std::shared_ptr<AgentManager> agent_manager) {
   agent_manager_ = agent_manager;
 }
 
+void WorkerPool::PopWorkerCallbackExecution(const PopWorkerCallback &callback,
+                                            std::shared_ptr<WorkerInterface> worker,
+                                            Status status) {
+  if (callback) {
+    // Call back this function asynchronously if needed. Such as
```

Comment:
I don't understand this comment. The code seems to be calling the callback synchronously?

Reply:
This method also seems to be doing very little, could we just inline it wherever it gets called?